embark.el

```diff
@@ -2070,12 +2070,16 @@ minibuffer before executing the action."
                             (embark--run-action-hooks embark-pre-action-hooks
                                                       action target quit)
                             (minibuffer-with-setup-hook inject
-                              ;; pacify commands that use (this-command-keys)
-                              (when (= (length (this-command-keys)) 0)
+                              ;; HACK work around `browse-url-interactive-arg'
+                              ;; expecting a non-empty `this-command-keys'
+                              ;; output.
+                              (when (and (string-match-p "browse-url"
```

Comment:
Symbols are much easier to compare for equality, I'd use (eq action 'browse-url).

Reply:
Okay. Just changed it to that.